test.js

```diff
@@ -2,24 +2,27 @@
 var assert = require('assert');
 var semverRegex = require('./');
 
-var fixture = [
-	'0.0.0',
-	'0.10.0',
-	'v1.0.0',
-	'0.0.0-foo',
-	'1.2.3-4',
-	'2.7.2+asdf',
-	'1.2.3-a.b.c.10.d.5',
-	'2.7.2-foo+bar',
-	'1.2.3-alpha.10.beta.0',
-	'1.2.3-alpha.10.beta.0+build.unicorn.rainbow',
-	'foo 0.0.0 bar 0.0.0'
-];
+var fixture = {
+	'0.0.0': ['0.0.0'],
+	'0.10.0': ['0.10.0'],
+	// fails due to https://github.com/sindresorhus/semver-regex/issues/7
+	// 'v1.0.0': ['1.0.0'],
+	'0.0.0-foo': ['0.0.0-foo'],
+	'1.2.3-4': ['1.2.3-4'],
+	'2.7.2+asdf': ['2.7.2+asdf'],
+	'1.2.3-a.b.c.10.d.5': ['1.2.3-a.b.c.10.d.5'],
+	'2.7.2-foo+bar': ['2.7.2-foo+bar'],
+	'1.2.3-alpha.10.beta.0': ['1.2.3-alpha.10.beta.0'],
+	'1.2.3-alpha.10.beta.0+build.unicorn.rainbow': ['1.2.3-alpha.10.beta.0+build.unicorn.rainbow'],
+	'foo 0.0.0 bar 0.1.1': ['0.0.0', '0.1.1']
```

Comment:
Instead of changing all the fixtures, can you instead just move this into a separate test?